docs/synthetics/status-pages.md

```diff
@@ -0,0 +1,37 @@
+title: Status Pages
+description: Communicate the status of your services to your customers using status pages
+
+Using Sematext Synthetics you can create public status pages to share the status and response time of your services to your customers.
```

Comment:
I think we need to be explicit about these pages not really being public unless one knows their full URL that includes a secret token and that these status pages are suitable for internal usage.  Maybe a short list of use cases is in order here.

Reply:
Done. Also added a few use cases